src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/ChromeOptions.kt

```diff
@@ -0,0 +1,54 @@
+package com.epam.drill.test.agent.instrumentation.http.selenium
+
+import com.epam.drill.test.agent.*
+import com.epam.drill.test.agent.instrumentation.*
+import com.epam.drill.test.agent.js.*
+import javassist.*
+import java.io.*
+import java.security.*
+
+class ChromeOptions : Strategy() {
+    private val extensionFile: String
+    private val dispatcherUrl = AgentConfig.dispatcherUrl().orEmpty()
+    private val adminUrl = AgentConfig.adminAddress().orEmpty()
+    private val agentId = AgentConfig.agentId().orEmpty()
+    private val groupId = AgentConfig.groupId().orEmpty()
+    private val agentAddress = AgentConfig.agentUrl().orEmpty()
+    private val clientId = ExtensionDispatcher.clientId().orEmpty()
+
+    private val chrome = "org.openqa.selenium.chrome.ChromeOptions"
+
+    init {
+        val specifiedPath = AgentConfig.extensionUrl().orEmpty()
+        extensionFile = if (specifiedPath.isBlank()) {
+            ExtensionLoader().loadExtension()
+            val tmpDir = System.getProperty("java.io.tmpdir")
+            File("$tmpDir/$JS_EXTENSION_NAME").absolutePath
+        } else specifiedPath
+    }
+
+    override fun permit(ctClass: CtClass): Boolean {
+        return ctClass.name == chrome
+    }
+
+    override fun instrument(
+        ctClass: CtClass,
+        pool: ClassPool,
+        classLoader: ClassLoader?,
+        protectionDomain: ProtectionDomain?
+    ): ByteArray? {
+
+        val agentOrGroup = if (groupId.isBlank()) "agentid=$agentId" else "groupid=$groupId"
+        val args = "new-window drill.autotesting.params?adminurl=$adminUrl&dispatcherUrl=$dispatcherUrl" +
+                "&$agentOrGroup&clientid=$clientId&agenturl=$agentAddress"
```

Comment:
is that special made in lower case? agentid, groupid, adminurl

Reply:
Roman said that there is a reason to use lowercase. So I changed dispatcherUrl to lowercase.